parasol_cpu/src/proc/assembly.rs

```diff
@@ -155,100 +291,91 @@ macro_rules! define_op {
 define_op! {
     IsaOp,
     DispatchIsaOp,
-    (Register, PtrRegister),
-
-    // Bind input
-    [BindReadOnly (dst dst, 1, PtrRegister) (meta id usize) (meta encrypted bool)],
-
-    // Bind output
-    [BindReadWrite (dst dst, 1, PtrRegister) (meta id usize) (meta encrypted bool)],
+    (Register),
 
     // Load
-    [Load (dst dst, 0, Register) (src src, 1, PtrRegister) (meta width u32)],
+    [0x02 Load (dst dst, 0, Register) (src src, 0, Register) (meta width, 7, u32)],
 
     // Load immediate
-    [LoadI (dst dst, 0, Register) (meta imm u128) (meta width u32)],
+    [0x03 LoadI (dst dst, 0, Register) (meta imm, 44, u128) (meta width, 7, u32)],
 
     // Store
-    [Store (src dst, 1, PtrRegister) (src src, 0, Register) (meta width u32)],
+    [0x04 Store (src dst, 0, Register) (src src, 0, Register) (meta width, 7, u32)],
```

Comment:
I assume the opcode numbers come from the compiler; potentially add a note on where to change them in that codebase.